wasm-host/src/main.rs

```diff
@@ -89,7 +89,7 @@ fn load_test_data(
     Ok((tx_json, lo_json, lh_json, l_json, nft_json))
 }
 
-fn main() {
+fn main() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
what is the benefit of returning the error from main? so that the CI can see something wrong?

Reply:
So if CI would pass either way, what is the benefit of returning error from main? easier to eye-ball?